src/components/blocks/ageGroupBlock.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { graphql, Link, useStaticQuery } from 'gatsby';
+import { BlockProps, BlockType } from '.';
+import { StrapiAgeGroup } from '../../../graphql-types';
+import { prependApiUrl } from '../../utils/helpers';
+import { findHeaderItemByTypeAndId } from '../../utils/navigation';
+import useNavigation from '../../hooks/navigation';
+import RichText from '../RichText';
+import { hexToRgba } from '../../utils/color';
+
+export interface AgeGroupBlockType extends BlockType {
+  title?: string;
+  ingress?: string;
+}
+
+const query = graphql`
```

Comment:
These queries could be all in one place, in queries folder. Makes finding and modifying easier I think.